### DISCUSSION TITLE
posts/clarification_of_why_i_am_disappointed_with_the_third_generation_of_wolf/

### DISCUSSION THREAD

**Discussion by giscus** [HalfSweet/Blog#22] [comment]
# posts/clarification_of_why_i_am_disappointed_with_the_third_generation_of_wolf/

在上文 一文中，我对于第三代郊狼的硬件部分进行了一些初步分析，因为没有详细的一些资料，因此在一些地方可能存在一些错误。在昨晚（2024-03-19）郊狼作者找到我，与我群群友进行了一番友好的长时间地交流，也解答了我对设计上的一些疑问。在本文中我们会逐步分析前文中提到的问题。 外观设计 在外观上，我质疑这个和二代的设计区别不大，有炒冷饭之嫌。 但是按照原...

https://cf.blog.halfsweet.cn/posts/clarification_of_why_i_am_disappointed_with_the_third_generation_of_wolf/

**Comment by Asanatsa** [comment]
好家伙，玩嵌入式的药娘😨

**另外PY32Duino的存储桶似乎炸了**

**Reply by HalfSweet** [reply]
国内的oss服务太贵了，而且这个架构感觉有问题准备重构

如果现在想用的话可以先用GitHub源（逃
